gap/xmltree.gi

```diff
@@ -34,28 +34,5 @@ InstallGlobalFunction( OMParseXmlObj, function ( node )
     return obj;
 end );
 
-
-
-
-InstallGlobalFunction( OMgetObjectXMLTree, function ( string )
```

Comment:
Looks right to remove this. Note that it also appears in `init.g` and `read.g` and also in comments in SCSCP's `openmath.g` file.